ando/tools/generator/utils.py

```diff
@@ -11,3 +11,16 @@ def save_tsv(dataframe, path_to_save):
 
     else:
         dataframe.to_csv(path_to_save, sep="\t", index=False)
+
+def save_json(dict, path_to_save):
+    if path.exists(path_to_save):
+        with open(path_to_save) as json_file:
+            data_existing = json.load(json_file)
+            for key_exist in data_existing.keys():
```

Comment:
I would put the actual merge of the dict in a separate function, just to keep the file handling separate from the merging.

Reply:
done